src/ctia/http/handler.clj

```diff
@@ -78,17 +80,23 @@
     (assert services->routes (str "Missing :services->routes for " (:entity entity)))
     (services->routes services-map)))
 
+(defn- entity-enabled?
+  [entity
+   {{:keys [enabled?]} :FeaturesService}]
+  (enabled? entity))
+
 (defmacro entity-routes
   [entities services-map]
   (let [gsm (gensym 'services-map)]
   `(let [~gsm ~services-map]
      (compojure.api.sweet/routes
       ~@(for [entity (remove :no-api?
                              (vals (eval entities)))]
-          `(context
-            ~(:route-context entity) []
-            :tags ~(:tags entity)
-            (entity->routes (~(:entity entity) entities) ~gsm)))))))
+          `(when (entity-enabled? ~(:entity entity) ~gsm)
+             (context
+                 ~(:route-context entity) []
+               :tags ~(:tags entity)
```

Comment:
I would hide routes in Swagger like this:
```suggestion
               :tags ~(:tags entity)
               :no-doc ~(:no-doc entity)
```

Reply:
This probably needs to be computed at runtime, so I'd imagine more like:

```clojure
   :no-doc (not (entity-enabled? ~(:entity entity) ~gsm))
```

Reply:
AFAIK, `:no-doc` is different, it only hides a route in Swagger (without disabling it)